### PR TITLE
Implement sort feature

### LINKED ISSUE #141: Add sort feature

### LINKED ISSUE #211: As a user, I want to sort my inventory by the fields I want, so that I can view my stocks easier.

### PR DESCRIPTION
**Pull request by AudreyFelicio**

### CODE REVIEW

**Review by Ashley-Lau — APPROVED** [review]
LGTM! Maybe an improvement is to allow user to sort according to ascending or descending order.

**Comment by AudreyFelicio** [comment]
Done implementing both ascending and descending order

File: src/main/java/seedu/stock/commons/util/SortUtil.java [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+        switch (field) {
+        case NAME:
+        case QUANTITY:
+        case SOURCE:
+        case LOCATION:
+            return field.toString().toLowerCase();
+        case SERIALNUMBER:
+            return "serial number";
+        default:
+            return "";
```
Comment (kaichaoang) [inline src/main/java/seedu/stock/commons/util/SortUtil.java:34]: Will this case ever be reached? If not, perhaps adding an assertion can be more defensive!

File: src/main/java/seedu/stock/commons/util/SortUtil.java [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+                String serialNumberB = b.getSerialNumber().toString();
+                return serialNumberA.compareTo(serialNumberB);
+            }
+        };
+    }
+
+    private static Comparator<Stock> generateQuantityComparator() {
+        return new Comparator<Stock>() {
+            @Override
+            public int compare(Stock a, Stock b) {
```
Comment (kaichaoang) [inline src/main/java/seedu/stock/commons/util/SortUtil.java:123]: Using BigInteger may slow the program down!

**Review by kaichaoang — APPROVED** [review]
LGTM!

**Pull request merged**